### PR TITLE
#861n6f0z0: Username to failed syncs

### PR DESCRIPTION
**Pull request by luxusko**
[#861n6f0z0](https://app.clickup.com/t/861n6f0z0)

### CODE REVIEW

File: server/mergin/auth/db_events.py [hunk 1]
```diff
@@ -5,10 +5,22 @@
 from .app import force_delete_user
 from .. import db
 from ..auth.models import User
+from mergin.sync.models import SyncFailuresHistory
+
+
+def remove_from_SyncFailuresHistory(user: User):
```
Comment (varmar05) [inline server/mergin/auth/db_events.py:11]: please convert to lower case and add doc string

**Review by varmar05 — CHANGES_REQUESTED** [review]
@luxusko could you update tests?

**Comment by varmar05** [comment]
@tomasMizera is there any reason why we do not wan't to go with user id as FK and cascade delete?
I imagine we only analyse such data using some BI.

**Comment by tomasMizera** [comment]
> @tomasMizera is there any reason why we do not wan't to go with user id as FK and cascade delete? I imagine we only analyse such data using some BI.

Well, it would work with CE, but I am afraid other versions could not use this

**Pull request merged**